pages/wizard/regions/task_step_bar.py

```diff
@@ -24,6 +24,22 @@
         'class': 'SetupType',
         'library': 'pages.wizard.rhev.setup_type',
     },
+    'Engine': {
+        'class': 'Engine',
+        'library': 'pages.wizard.rhev.engine',
+    },
+    'Hypervisor': {
+        'class': 'Hypervisor',
+        'library': 'pages.wizard.rhev.hypervisor',
+    },
+    'Configuration': {
```

Comment:
The library is using the text from the vertical task bar to do the lookup.   What we need to do is add the context of the current deployment step name, and then we can namespace the keys like you have suggested.   I have done this here:

https://github.com/RedHatQE/fusor-qe/pull/36/commits/5305472ca0541de81a783f6d76bc114ca2798e73